core/domain/topic_domain.py

```diff
@@ -153,7 +273,11 @@ def to_dict(self):
             'canonical_story_ids': self.canonical_story_ids,
             'additional_story_ids': self.additional_story_ids,
             'language_code': self.language_code,
-            'skill_ids': self.skill_ids,
+            'subtopic_schema_version': self.subtopic_schema_version,
```

Comment:
Consider having these be in the same order as lines 250-260 or so.

Reply:
Done.